pages/crowdinintegration.md

```diff
@@ -1,9 +1,12 @@
 [Google Doc](https://docs.google.com/document/d/1gsDmBSq5Tu1G_Ei1CfTRUuvvwpAWUPhHYr2UYESwISI/edit?usp=sharing)
 # Crowdin Integration
-##Objectives:
+
+## Objectives:
+
 There are two objectives for this assignment:
-* 1- Provide exactly one screenshot for the strings in BeLL app, and tag it.
-* 2- Write good description of the strings in Context.
+
+* 1 - Provide exactly one screenshot for the strings in BeLL app, and tag it.
+* 2 - Write good description of the strings in Context.
```

Comment:
I think just using an ordered list here would be more readable.  [Check out the markdown documentation](https://daringfireball.net/projects/markdown/syntax#list) if you need more info.